Index: python-novaclient/novaclient/v1_1/servers.py
===================================================================
--- python-novaclient.orig/novaclient/v1_1/servers.py	2011-12-02 18:01:39.698190398 +0000
+++ python-novaclient/novaclient/v1_1/servers.py	2011-12-02 18:01:09.934015723 +0000
@@ -282,37 +282,37 @@
         """
         Pause the server.
         """
-        self._action('pause', server, None)
+        self.api.client.post('/servers/%s/pause' % base.getid(server))
 
     def unpause(self, server):
         """
         Unpause the server.
         """
-        self._action('unpause', server, None)
+        self.api.client.post('/servers/%s/unpause' % base.getid(server))
 
     def suspend(self, server):
         """
         Suspend the server.
         """
-        self._action('suspend', server, None)
+        self.api.client.post('/servers/%s/suspend' % base.getid(server))
 
     def resume(self, server):
         """
         Resume the server.
         """
-        self._action('resume', server, None)
+        self.api.client.post('/servers/%s/resume' % base.getid(server))
 
     def rescue(self, server):
         """
         Rescue the server.
         """
-        self._action('rescue', server, None)
+        self.api.client.post('/servers/%s/rescue' % base.getid(server))
 
     def unrescue(self, server):
         """
         Unrescue the server.
         """
-        self._action('unrescue', server, None)
+        self.api.client.post('/servers/%s/unrescue' % base.getid(server))
 
     def diagnostics(self, server):
         """Retrieve server diagnostics."""
